server/utils.js

```diff
@@ -0,0 +1,31 @@
+splitData = (result) => {
```

Comment:
I like how you're doing a lot of your organization on the backend. This is something I probably need to refactor for mine to do as well.